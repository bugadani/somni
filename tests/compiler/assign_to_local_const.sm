fn main() {
    const C = 10;
    // This should fail, as C is a constant and cannot be modified.
    C = 20; // Should cause a compile error
}