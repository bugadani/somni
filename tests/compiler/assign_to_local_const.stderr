Compile error
 ---> at line 4 column 5
  |
4 |     C = 20; // Should cause a compile error
  |     ^^^^^^ Cannot assign to a constant